utils/https.h

```diff
@@ -41,10 +49,25 @@ int https_logger_release();
  * @param[in] api API path for POST request to HTTP(S) server, i.e "transaction/". It must be in string.
  * @param[in] msg Message to send
  * @param[in] ssl_seed Seed for ssl connection
+ * @param[out] s_res The message body of HTTP(S) response
  *
  * @return #status_t
  */
-status_t send_https_msg(char const *host, char const *port, char const *api, const char *msg, const char *ssl_seed);
+status_t send_https_msg(char const *host, char const *port, char const *api, const char *msg, const char *ssl_seed,
```

Comment:
It is definitely inefficient to pass 6 parameters for one function. Would you consider to introduce a dedicated structure for preserving these parameters?